### PR TITLE
Add helm chart for clustered Integrator profile

### LINKED ISSUE #78: Create Helm charts for clustered Integrator profile
**Description:**
Helm charts should be created for the deployment of clustered Integrator profile.

**Suggested Labels:**
Type/Task

**Suggested Assignees:**

**Affected Product Version:**
WSO2 Enterprise Integrator v6.2.x

**OS, DB, other environment details and versions:**    

**Steps to reproduce:**

**Related Issues:**

### PR DESCRIPTION
**Pull request by KavinduZoysa**
## Purpose
Create Helm chart for the deployment of clustered Integrator profile. Resolves #78 

## Goals
Give the ability to deploy clustered Integrator profile using Helm.

## Approach
Create Helm charts for clustered Integrator profile.

## User stories
N/A

## Release note
Create Helm chart for clustered Integrator profile.

## Documentation
N/A

## Training
N/A

## Certification
N/A

## Marketing
N/A

## Automation tests
N/A

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

## Samples
N/A

## Related PRs
N/A

## Migrations (if applicable)
N/A

## Test environment
Helm v2.8.2
Tiller v2.9.1
 
## Learning
https://docs.helm.sh/using_helm/#quickstart
https://kubernetes.github.io/ingress-nginx/deploy/#using-helm
https://godoc.org/text/template
https://godoc.org/github.com/Masterminds/sprig

### CODE REVIEW

File: helm/scalable-integrator/README.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+* In order to use these Kubernetes resources, you will need an active [Free Trial Subscription](https://wso2.com/free-trial-subscription)
+from WSO2 since the referring Docker images hosted at docker.wso2.com contains the latest updates and fixes for WSO2 Enterprise Integrator.
+You can sign up for a Free Trial Subscription [here](https://wso2.com/free-trial-subscription).<br><br>
+
+* Install [Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git), [Helm](https://github.com/kubernetes/helm/blob/master/docs/install.md)
+(and Tiller) and [Kubernetes client](https://kubernetes.io/docs/tasks/tools/install-kubectl/) in order to run the 
+steps provided in the following quick start guide.<br><br>
+
+* Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/). This can
```
Comment (chirangaalwis) [inline helm/scalable-integrator/README.md]: @KavinduZoysa IMO it is better to leave this as `Install [NGINX Ingress Controller](https://kubernetes.github.io/ingress-nginx/deploy/)`.

File: helm/scalable-integrator/scalable-integrator-conf/Chart.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (msmshariq) [inline helm/scalable-integrator/scalable-integrator-conf/Chart.yaml]: Shouldn't the name and description have something to indicate that this chart is for WSO2 EI ?

File: helm/scalable-integrator/scalable-integrator-deployment/Chart.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: v1
+appVersion: "1.0"
+description: A Helm chart for Kubernetes
```
Comment (msmshariq) [inline helm/scalable-integrator/scalable-integrator-deployment/Chart.yaml]: Here also we need to indicate the chart is for WSO2 EI Integrator profile etc.

File: helm/scalable-integrator/scalable-integrator-conf/Chart.yaml (hunk 2 above)
Comment (KavinduZoysa) [reply helm/scalable-integrator/scalable-integrator-conf/Chart.yaml]: Changed the name

File: helm/scalable-integrator/scalable-integrator-deployment/Chart.yaml (hunk 3 above)
Comment (KavinduZoysa) [reply helm/scalable-integrator/scalable-integrator-deployment/Chart.yaml]: Changed the name

**Pull request merged**